pyproject.toml

```
@@ -0,0 +1,21 @@
+# As per https://github.com/pypa/setuptools/blob/main/docs/userguide/quickstart.rst
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+
+[options]
+dependencies = [
+    "amiga-brain-api @ git+https://github.com/farm-ng/amiga-brain-api@canbus-packet",
```

After merging https://github.com/farm-ng/amiga-brain-api/pull/18
```suggestion
    "amiga-brain-api @ git+https://github.com/farm-ng/amiga-brain-api@main",
```